compiler/compile/OMRCompilation.cpp

```diff
@@ -930,13 +942,7 @@ int32_t OMR::Compilation::compile()
 
    if (self()->isOptServer())
       {
-      // Temporarily exclude PPC due to perf regression
-      if( (self()->getMethodHotness() <= warm))
-         {
-         if (!TR::Compiler->target.cpu.isPower())
-            TR::Options::getCmdLineOptions()->setOption(TR_DisableInternalPointers);
-         }
-      self()->getOptions()->setOption(TR_DisablePartialInlining);
+         self()->setOption(TR_DisablePartialInlining);
```

Comment:
Please fix indenting to be style consistent.

Reply:
done